src/renderer/store/modules/boards.js

```diff
@@ -22,9 +22,11 @@ const mutations = {
 }
 
 const actions = {
-  async fetchBoards ({ commit }) {
+  async fetchBoards ({ commit, dispatch }) {
+    dispatch('wait/start', 'boardsLoading', { root: true })
```

Comment:
Maybe we should try the [waitFor decorator](https://github.com/f/vue-wait#waitforloader-string-func-function-forcesync--false)?